docs/source/getting-started/path-variables.rst

```diff
@@ -24,48 +24,31 @@ If you have downloaded and installed the SDK using our `Windows installer <https
 
 Mac OS
 ******
-We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Mac OS with ``zsh`` shell.
-If you are using ``bash`` all of the instructions are quite similar, except that you will be doing all of the changes
-in the ``.bash_profile`` file.
 
-Setting the JAVA_HOME variable
-==============================
-Run the following command in your terminal::
+Open a terminal and run the following commands. Typos are a big problem here so copy/paste one line at a time if possible. None of these should produce any output on success::
 
         echo 'export JAVA_HOME="$(/usr/libexec/java_home)"' >> ~/.zprofile
-
-Setting the PATH variable
-=========================
-The installer will ask you and set the ``PATH`` variable for you. If you want to set your ``PATH`` variable
-manually instead, run the following command in your terminal::
-
+        echo 'export JAVA_HOME="$(/usr/libexec/java_home)"' >> ~/.bash_profile
         echo 'export PATH="$HOME/.daml/bin:$PATH"' >> ~/.zprofile
+        echo 'export PATH="$HOME/.daml/bin:$PATH"' >> ~/.bash_profile
 
-Verifying the changes
-=====================
-
-In order for the changes to take effect you will need to restart your computer, or, if you're using
-the macOS Terminal app, you only need to quit the Terminal app (Command+Q in the Terminal window) and
-reopen it. Afterward, please follow the instructions below to verify that everything was set
-up correctly.
-
-Please verify the JAVA_HOME variable by running::
+This will update the configuration for future, newly opened terminals, but will not affect any exsting one. To test the configuration of ``JAVA_HOME``, open a new terminal and run::
```

Comment:
Read the forum thread on the installation instructions. Found this a bit more confusing as it's mixing instructions for both `zsh` and `bash`. Would maybe make it clearer at the beginning that current instructions are for `zsh` 

Reply:
I would expect people who know the difference between Bash and Zsh to not need this page at all. Under this assumption, I don't see a point in mentioning either; following these instructions will work for both without the user having to know about either, nor which one they're using.

Reply:
> I would expect people who know the difference between Bash and Zsh to not need this page at all.

Think this is a pretty big assumption. I don't know the exact diff, just that there is one and that I had to go through different processes when setting up the `PATH` and `JAVA_HOME` variables. 

@moritzkiefer-da also mentioned that there are a number of shells out there. Being explicit about our instructions does help.

Reply:
Again, I very much expect that people who have installed a non-default shell know what they are doing enough to:
1. Recognize that all we're doing here is set the values of two env vars, and
2. Know how to set up env vars in their shell.
In my view these instructions are strictly for people who don't know how a shell works.

I may totally be wrong about this, which is what the review process is for. 🙂

Reply:
Being explicit about 1) the shell and 2) what they're setting up 3) helps the user look for answers in case they run into issues. Giving them just the commands leaves them dry.

1. We could be removing the bit on `bash`, as this might have caused the confusion. Our instructions are for `zsh` and we mention `bash` as an example. 

2. The command that helped krmmalik set up the `JAVA_PATH` is slightly different form the one that we currently have. He used 

`echo "export JAVA_HOME=`/usr/libexec/java_home`" >> ~/.profile    
. ~/.profile` 

while our instructions have 
`echo 'export JAVA_HOME="$(/usr/libexec/java_home)"' >> ~/.zprofile` 

We might wanna change to that one (not a `shell` expert I'm afraid)

Reply:
`$(...)` is the modern way of doing backticks. They're basically equivalent but `$(...)` can be nested, so most people favor it. It works in `bash` (and `zsh`, `ash`, etc.) but not `sh`.

Reply:
also not sure if adding shell profiles for possibly non-existing shells is a good thing to do.

Reply:
> also not sure if adding shell profiles for possibly non-existing shells is a good thing to do.

Adding shell profiles for unused shells is harmless, they are just text files that will never be read.

Reply:
@nemanja-da I've added a step to figure out which shell is running, and (hopefully) clarified which commands are for which shell.

Reply:
@garyverhaegen-da Love the `PATH` variable changes! Left a comment for the installation instructions.